### ISSUE TITLE
Can I load a midi file and get audio samples?

### ISSUE THREAD

**Issue by happyTonakai** [nwhitehead/pyfluidsynth#71]
I can play a MIDI file using the following example provided in the documentation. However, I'm looking for a way to convert MIDI files directly into audio samples.

```py
fs = fluidsynth.Synth()
fs.start()
sfid = fs.sfload("FluidR3_GM.sf2")

fs.play_midi_file("1080-c01.mid")
time.sleep(20)
```

The document includes an example of the `get_samples()` method, which generates a chord from scratch and gets the audio samples, but I'm interested in loading from a MIDI file directly. Is this functionality available, or is there a recommended approach to achieve this?

I managed to get the audio samples by calling `fs.get_samples(1024)` right after `fs.play_midi_file()` which is not an elegant way I think.

Thank you in advance!

**Comment by nwhitehead** [comment]
I took a look into this. I agree, calling the `get_samples` right after the `play_midi_file` was the current best way but not ideal since you already had to have called `start`.

I updated the code to fix that and added a couple examples. `test/test5.py` shows just playing a MIDI file, and `test/test6.py` shows generating 10 seconds of samples and then playing it outside of FluidSynth.

Let me know if the change works for you.
